files/tools/terraform_env_variable_helper.sh

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env bash
+# `set -e` is used below to ensure that all of the assignments in the script are made / that it runs in entirety
 set -e
 
+# usage: run the export in either line 8 or 9 before this script or edit lines 8 or 9 in this script before running
 # usage: source <path-to-file>TerraformEnvVariableAssignment.sh
 # note: select one of the following options for your Service Principal
 export YourSP="<yourSP>"
```

Comment:
This item should follow stand ENV variable formatting. All caps. The value should be: YOURSP. Also this line should be removed. The ENV should be required by the script of it fails.